src/common/utils/compare.ts

```diff
@@ -34,3 +36,29 @@ export function getCountiesArr(stateId: string) {
       };
     });
 }
+
+export function getDisclaimerCopy(
+  countiesArr: SummaryForCompare[],
+  stateName?: string,
+) {
+  const contactTracingData = countiesArr.filter(
+    (location: any) =>
+      location.metricsInfo.metrics[Metric.CONTACT_TRACING].level !==
+      Level.UNKNOWN,
+  );
+  const positiveTestData = countiesArr.filter(
+    (location: any) =>
+      location.metricsInfo.metrics[Metric.POSITIVE_TESTS].level !==
+      Level.UNKNOWN,
+  );
+  if (contactTracingData.length && positiveTestData.length) {
+    return 'Most states only report contact tracing at the state level.';
```

Comment:
Do we need the disclaimer when we do have data for contact tracing and positive tests?

Reply:
my thought was: i think there's currently only a handful of counties with contact tracing data (maybe just 1 county?), so if there _is_ a table with contact tracing, it will be one county out of many, in which case there should probably still be a disclaimer re: why the column is nearly empty

Reply:
but yeah, can nix it all together if that feels like it makes more sense

Reply:
but if all the counties have both contact tracing and positive test data, then this condition would be `true`,  I'm not sure if we need the disclaimer in that case.

Reply:
updates: new mocks nix dynamic copy, so this is all going away..